ethexe/ethereum/src/wvara/mod.rs

```diff
@@ -44,9 +44,18 @@ impl WVara {
         LocalAddress(*self.0.address().0)
     }
 
-    // TODO (breathx): handle events.
     pub async fn approve(&self, address: Address, value: u128) -> Result<H256> {
         let value = Uint::<256, 4>::from(value);
+
+        self._approve(address, value).await
+    }
+
+    pub async fn approve_all(&self, address: Address) -> Result<H256> {
+        self._approve(address, Uint::<256, 4>::MAX).await
```

Comment:
I think it can be just U256: https://docs.rs/alloy/latest/alloy/primitives/type.U256.html